README.md

```diff
@@ -1 +1,181 @@
-# Project-Review
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<div align="center">
+  <img src="images/Cizzla_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>My Personal Portifolio README Template</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+
+# 📖 My Personal Portifolio <a name="about-project"></a>
+
+
+
+**My Personal Portifolio** is a website that will showcase my bio information,skill and projects that I have done.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/default.asp">HTML</a></li>
+    <li><a href="https://www.w3schools.com/css/default.asp/">CSS</a></li>
+  </ul>
+</details>
+
+
+
+
+### Key Features <a name="key-features"></a>
+
+
+- **Portifolio**
+- **About me**
+- **Contact**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+<ul>
+    <li>Web Browser</li>
+    <li>Code Editor</li>
+    <li>Html Knowledge</li>
+    <li>Css Knowledge</li>
+    <li>Local file storage</li>
+</ul>
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Run this commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:iankiema/My-Personal-Portifolio.git
+```
+
+### Install
+
+Install this project with:
+
+Run this command:
+
+```sh
+  cd my-project
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+- Open index.html using live server extension or;
+
+Run this  command:
+
+```sh
+  start index.html
+```
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Ian Kiema**
+
+- GitHub: [@githubhandle](https://github.com/iankiema)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](www.linkedin.com/in/ian-kiema-73419779)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **About Me**
+- [ ] **Portifolio**
+- [ ] **Contact**
```

Comment:
You have already updated the features mentioned. I suggest you look for any other upcoming features for your project. 🧑‍💻. The new features could be adding projects to your portfolio. This will help the end user anticipate the new features.